### ISSUE TITLE
Backup issue

### ISSUE THREAD

**Issue by Joci64** [hoobs-org/HOOBS#1679]
Hi,
My old Hoobs Box would be replaced and I just wanted to restore the new Hoobs Box from my backup. Even though I have made a backup that included all plugins and bridges and worked before properly, but there are either devices or configuration in the new Box after restore. I have already done factory reset two times but it does not help. Moreover I am not able to install any plugins. I have already contacted info@hoobs.com , here is what they have responsed to me:
“ From the provided logs - the system is able to do a full restore from the backup.
However, once the restore is complete - it loads the bridge but prompts no plugins were installed. See below that I have highlighted.
The only way is to re-install and re-configure the plugins and also you can open a GitHub Issue for our dev team to take a look at your backup and understand the issue that has occurred.“

Should I really reinstall and configure all plugins and devices again, even though I have already done this work once and it has taken a lot of time?
Please tell me what additional information you need for helping me with this issue. 

I would be very glad to hearing from you soon.

Best Regards
Joci64

**Comment by BobbySlope** [comment]
I think the issue with the backup and restore was that your old box was on version 3 and the replacement box is already on version 4 and the backups are no longer compatible between versions 3/4
